.rubocop.yml

```diff
@@ -1468,9 +1468,18 @@ Style/MapToHash:
   Enabled: false
 
 Style/MethodCallWithArgsParentheses:
-  Enabled: false
+  Enabled: true
+  IgnoredMethods:
+    - puts
+    - require
+    - raise
+    - include
+    - fail
+    - print
+    - add_dependency
+    - add_development_dependency
+  AllowedPatterns: [^assert, ^refute]
```

Comment:
This was my best guess at the Ruby methods we'd want to ignore/allow args to be passed without parentheses. I am open to suggestions to change this list! 

All changes outside .rubocop.yml were performed through `bundle exec rubocop -a`